docsets/FileMaker/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "FileMaker",
-    "version": "17.0.1",
+    "version": "18.0.0",
     "archive": "FileMaker.tgz",
```

Comment:
You need to update the main archive as well.

Reply:
Sorry about that. I updated the main archive, rebased, and squashed to a single commit.